Fixed bug with minor peak (#30398)
mantidproject/mantid: docs/source/release/v6.0.0/muon.rst

New Features
############
- Added a new XrayAbsorptionCorrection algorithm. See :ref:`XrayAbsorptionCorrection <algm-XrayAbsorptionCorrection>`

> - Fixed bug in GUI where minor peaks wouldn't be plotted on new detector when checkbox clicked.

Reviewer: ```suggestion
- Fixed bug in GUI where minor peaks wouldn't be added to a new detector subplot
```
